src/mainboard/emulation/qemu-q35/vboot-rwa-8M.fmd

```diff
@@ -2,6 +2,7 @@ FLASH@0xff800000 0x800000 {
 	SI_BIOS 0x800000 {
 		RW_SECTION_A 0x3c0000 {
 			VBLOCK_A 0x10000
+			BOOTSPLASH(CBFS) 0x40000
```

Comment:
I think this should be outside of RW_SECTION_A

Reply:
You're right, thanks.

https://github.com/Dasharo/coreboot/compare/db40b9c55555873c1807d1c7e7babd45b1bba380..bd294ab1d22b4010fafa0120e2eda26964316096